modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/TableKeyDetails/index.js

```diff
@@ -0,0 +1,300 @@
+/**
```

Comment:
I noticed that are some HTML components here quite similar. It's possible to merge (and generalize) them in some function and use it?

I'm thinking because this component has a lot lines of code, **if possible**, let's try simplify.